dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/FileUtils.java

```diff
@@ -289,6 +291,29 @@ public static void setFileTo755(File file) throws IOException {
         }
     }
 
+    public static void createDirectoryWith775(@NonNull Path path) throws IOException {
+        if (path.toFile().exists()) {
+            return;
+        }
+        if (OSUtils.isWindows()) {
+            Files.createDirectories(path);
+        } else {
```

Comment:
I think we can remove this since we don't support running on windows.

Reply:
+1

Reply:
Some task e.g. sql task can run on windows, user can use winfows to debug.

Reply:
Okay.